src/BenchmarkDotNet.Annotations/Jobs/RuntimeMoniker.cs

```diff
@@ -115,6 +115,11 @@ public enum RuntimeMoniker
         /// </summary>
         Net90,
 
+        /// <summary>
+        /// .NET 10.0
+        /// </summary>
+        Net100,
```

Comment:
Should we start adding `_` here?
```suggestion
        Net10_0,
```

Reply:
I think we should really consider moving away from these strong type names. This is too much ceremony to issue a nice message. If the underlying SDK doesn't support the TFM, it issues a very reasonable, comprehensive and googleable error message. For the other use-case, consider string parameter `[SupportedFramework("netX.Y")]` rather than these unscalable abstractions which we can't get rid of even after the TFM is rendered unsupported by rest of the ecosystem.

Reply:
Added underscore. However, the way SDK (and other parts of ecosystem) is handling these validation is they rely on common implementation in the SDK. If the underlying SDK doesn't support the TFM, it will report the error. Then there is the new concept of ["torn SDK"](https://github.com/dotnet/sdk/blob/main/documentation/general/torn-sdk.md) in .NET 9, which makes the tooling download the required SDK. For NativeAOT, LLVMAOT, Mono runtime etc. there are flags like `--aot` available on command line (e.g. `dotnet new console --aot`).

BDN has gone in other direction with these dedicated monikers, `nativeaot9.0` etc., which IMHO is an unnecessary maintenance headache, one we can easily avoid by using stringly arguments.

Reply:
Added underscore. However, the way SDK (and other parts of ecosystem) is handling these validations is they rely on common implementation in the SDK. If the underlying SDK doesn't support the TFM, it will report the error. Then there is the new concept of ["torn SDK"](https://github.com/dotnet/sdk/blob/main/documentation/general/torn-sdk.md) in .NET 9, which makes the tooling download the required SDK. For NativeAOT, LLVMAOT, Mono runtime etc. there are flags like `--aot` available on command line (e.g. `dotnet new console --aot`).

BDN has gone in other direction with these dedicated monikers, `nativeaot9.0` etc., which IMHO is an unnecessary maintenance headache, one we can easily avoid by using stringly arguments.

Reply:
I don't disagree. Feel free to open an issue for suggestions on improvements.